lib/livebook/storage/ets.ex

```diff
@@ -98,13 +109,38 @@ defmodule Livebook.Storage.Ets do
 
     :ets.insert(table, attributes)
 
+    :ok = save_to_file(state)
+
     {:reply, :ok, state}
   end
 
   @impl GenServer
   def handle_call({:delete, namespace, entity_id}, _from, %{table: table} = state) do
     :ets.delete(table, {namespace, entity_id})
 
+    :ok = save_to_file(state)
+
     {:reply, :ok, state}
   end
+
+
+  defp load_or_create_table(file_path) do
+    if File.exists?(file_path) do
+      {:ok, tab} = :ets.file2tab(String.to_charlist(file_path))
```

Comment:
If it fails for some reason, we should default to the new table :)

Reply:
Yeah, you can skip the `File.exists?` check. Instead try to load the file and match on `{:error, _}`.